libsolidity/formal/Predicate.cpp

```diff
@@ -187,12 +189,21 @@ string Predicate::formatSummaryCall(vector<smtutil::Expression> const& _args) co
 {
 	solAssert(isSummary(), "");
 
-	if (auto contract = programContract())
-		return contract->name() + ".constructor()";
-
 	if (auto funCall = programFunctionCall())
 		return funCall->location().text();
 
+	auto txVars = readTxVars(_args.at(4));
+	string value;
+	if (auto v = txVars.at("msg.value"))
+	{
+		bigint x(*v);
+		if (x > 0)
+			value = "{ value: " + *v + " }";
+	}
+
+	if (auto contract = programContract())
+		return contract->name() + ".constructor()" + value;
+
```

Comment:
Moving the comments on line 212 to 194 would help with understanding `_args.at(4)`.